gh-actions-scripts/build_installer_helm_chart.sh

```diff
@@ -11,6 +11,9 @@ BASE_PATH=installer/manifests
 helm repo add nats https://nats-io.github.io/k8s/helm/charts/
 helm dependency build ${BASE_PATH}/keptn/charts/control-plane
 
+# replace "appVersion: latest" with "appVersion: $VERSION" in all Chart.yaml files
+find -name Chart.yaml -exec sed -i -- "s/appVersion: latest/appVersion: ${VERSION}/g" {} \;
```

Comment:
Do we also need to properly set the `version`?

Reply:
I can add that too, if wanted/needed. This would be the version of the chart itself, right?

Reply:
Fixed.

Reply:
Yes right. It is the version of the chart.

Reply:
Until now, the appVersion always matches the chart version.